routemaster/feeds.py

```diff
@@ -20,6 +21,11 @@ class FeedNotFetched(Exception):
     pass
 
 
+@functools.lru_cache()
+def _get_feed_session():
+    return requests.Session()
```

Comment:
These aren't thread-safe: I think this should be thread-local.

Reply:
ba95b30